From f7fcc06097d862ca3bc6840b550a53e0b02f139f Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyfox@gentoo.org>
Date: Fri, 22 Jul 2011 12:36:06 +0300
Subject: [PATCH] fix build failure against ghc-7.2

    src/Plugins/Monitors/Top.hs:140:17:
        Illegal bang-pattern (use -XBangPatterns):
        ! r

Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>
---
 src/Plugins/Monitors/Top.hs |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/Plugins/Monitors/Top.hs b/src/Plugins/Monitors/Top.hs
index e45210c..6001164 100644
--- a/src/Plugins/Monitors/Top.hs
+++ b/src/Plugins/Monitors/Top.hs
@@ -13,6 +13,7 @@
 -----------------------------------------------------------------------------
 
 {-# LANGUAGE ForeignFunctionInterface #-}
+{-# LANGUAGE BangPatterns #-}
 
 module Plugins.Monitors.Top (startTop, topMemConfig, runTopMem) where
 
-- 
1.7.3.4

